server/src/test/java/org/elasticsearch/index/mapper/DynamicTemplateTests.java

```diff
@@ -40,7 +40,7 @@ public void testParseUnknownParam() throws Exception {
         templateDef.put("random_param", "random_value");
 
         IllegalArgumentException e = expectThrows(IllegalArgumentException.class,
-                () -> DynamicTemplate.parse("my_template", templateDef, Version.V_6_0_0_alpha1));
+                () -> DynamicTemplate.parse("my_template", templateDef, Version.V_7_0_0));
```

Comment:
I think all these 7.0 references can be `Version.CURRENT.minimumIndexCompatibilityVersion()`